SpreadCheetah/Spreadsheet.cs

```diff
@@ -60,15 +60,15 @@ private Spreadsheet(ZipArchive archive, CompressionLevel compressionLevel, int b
         var archive = new ZipArchive(stream, ZipArchiveMode.Create, true);
         var bufferSize = options?.BufferSize ?? SpreadCheetahOptions.DefaultBufferSize;
         var compressionLevel = GetCompressionLevel(options?.CompressionLevel ?? SpreadCheetahOptions.DefaultCompressionLevel);
-        var defaultDateTimeNumberFormat = options is null ? NumberFormats.DateTimeSortable : options.DefaultDateTimeNumberFormat;
+        var defaultDateTimeFormat = options is null ? new SpreadCheetahOptions().DefaultDateTimeFormat : options.DefaultDateTimeFormat;
```

Comment:
`new SpreadCheetahOptions()` - here is possibly an unnecessary allocation that I would like to avoid.

Reply:
I would also like to avoid duplication of the default, to avoid it getting out of sync. I've added the (slightly clunky named DefaultDefaultDateTimeFormat) static value to match the style of the other defaults there.

Reply:
Yeah I agree the name is a bit clunky. Can we make the static field internal instead of public?

Reply:
Done. It didn't need to be public, I just followed the existing example defaults.